crates/bevy_render/src/texture/mod.rs

```diff
@@ -170,8 +174,176 @@ pub trait BevyDefault {
+
+        match self {
+            ViewTargetFormat::R16Float
+            | ViewTargetFormat::Rg16Float
+            | ViewTargetFormat::Rgba16Float
+            | ViewTargetFormat::Rb11b10Float => true,
+            ViewTargetFormat::R8Unorm
+            | ViewTargetFormat::Rg8Unorm
+            | ViewTargetFormat::Rgba8Unorm
+            | ViewTargetFormat::Bgra8Unorm
+            | ViewTargetFormat::Rgba8UnormSrgb
+            | ViewTargetFormat::Bgra8UnormSrgb
+            | ViewTargetFormat::Rgb10a2Unorm => false,
+        }
+    }
+}
+
+impl From<ViewTargetFormat> for TextureFormat {
+    fn from(value: ViewTargetFormat) -> Self {
+        match value {
+            ViewTargetFormat::Rgba16Float => TextureFormat::Rgba16Float,
+            ViewTargetFormat::Rb11b10Float => TextureFormat::Rg11b10Float,
+            ViewTargetFormat::Rgba8UnormSrgb => TextureFormat::Rgba8UnormSrgb,
+            ViewTargetFormat::Rgba8Unorm => TextureFormat::Rgba8Unorm,
+            ViewTargetFormat::R8Unorm => TextureFormat::R8Unorm,
+            ViewTargetFormat::Rg8Unorm => TextureFormat::Rg8Unorm,
+            ViewTargetFormat::Bgra8Unorm => TextureFormat::Bgra8Unorm,
+            ViewTargetFormat::Bgra8UnormSrgb => TextureFormat::Bgra8UnormSrgb,
+            ViewTargetFormat::R16Float => TextureFormat::R16Float,
+            ViewTargetFormat::Rg16Float => TextureFormat::Rg16Float,
+            ViewTargetFormat::Rgb10a2Unorm => TextureFormat::Rgb10a2Unorm,
+        }
+    }
+}
+
+/// The error type returned when converting a [`TextureFormat`] into [`ViewTargetFormat`] fails.
+#[derive(Debug, Clone, Copy)]
+pub struct TryFromTextureFormatError {}
+
+impl Display for TryFromTextureFormatError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_str("this texture type can't be rendered to")
+    }
+}
+
+impl TryFrom<TextureFormat> for ViewTargetFormat {
+    type Error = TryFromTextureFormatError;
+
+    fn try_from(value: TextureFormat) -> Result<Self, Self::Error> {
+        match value {
+            TextureFormat::Rgba16Float => Ok(ViewTargetFormat::Rgba16Float),
+            TextureFormat::Rg11b10Float => Ok(ViewTargetFormat::Rb11b10Float),
+            TextureFormat::Rgba8UnormSrgb => Ok(ViewTargetFormat::Rgba8UnormSrgb),
+            TextureFormat::Rgba8Unorm => Ok(ViewTargetFormat::Rgba8Unorm),
+            TextureFormat::R8Unorm => Ok(ViewTargetFormat::R8Unorm),
+            TextureFormat::Rg8Unorm => Ok(ViewTargetFormat::Rg8Unorm),
+            TextureFormat::Bgra8Unorm => Ok(ViewTargetFormat::Bgra8Unorm),
+            TextureFormat::Bgra8UnormSrgb => Ok(ViewTargetFormat::Bgra8UnormSrgb),
+            TextureFormat::R16Float => Ok(ViewTargetFormat::R16Float),
+            TextureFormat::Rg16Float => Ok(ViewTargetFormat::Rg16Float),
+            TextureFormat::Rgb10a2Unorm => Ok(ViewTargetFormat::Rgb10a2Unorm),
+            _ => Err(TryFromTextureFormatError {}),
```

Comment:
```suggestion
            _ => Err(TryFromTextureFormatError),
```

Reply:
This exists so I can batch commit it with the `TryFromTextureFormatError` suggestion through the GitHub ui.